includes/functions.php

```diff
@@ -13,3 +15,268 @@
+			'fields' => 'names',
+		) );
+
+		if ( is_wp_error( $locations ) || empty( $locations ) ) {
+			$location = '--';
+		} else {
+			$location = $locations[0];
+		}
+
+		$output .= '<div class="m1 mt3 mb2"><div class="' . esc_html( $popular_classes[ $index ] ) . ' mb1">
+			<div class="relative travel-results-result">
+				<a class="travel-results-result-link block relative" href="' . esc_url( get_the_permalink( $adventure->ID ) ) . '">
+					<amp-img class="block rounded" width="346" height="200" noloading="" src="' . esc_url( $img_src ) . '" srcset="' . esc_attr( $img_srcset ) . '"></amp-img>
+				</a>
+			</div>
+		</div>
+		<div class="h2 line-height-2 mb1">
+			<span class="travel-results-result-text">' . esc_html( get_the_title( $adventure->ID ) ) . '</span>
+			<span class="travel-results-result-subtext h3">•</span>
+			<span class="travel-results-result-subtext h3">$&nbsp;</span><span class="black bold">' . esc_html( $price ) . '</span>
+		</div>
+		<div class="h4 line-height-2">
+			<div class="inline-block relative mr1 h3 line-height-2">
+				<div class="travel-results-result-stars green">';
+
+		for ( $i = 0; $i < $rating; $i++ ) {
+			$output .= '★';
+		}
+
+		$output .= '</div>
+			</div>
+			<span class="travel-results-result-subtext mr1">' .
+			/* translators: %d: The number of reviews */
+			sprintf( esc_html__( '%d Reviews', 'travel' ), esc_html( $reviews->approved ) ) . '</span>
+				<span class="travel-results-result-subtext"><svg class="travel-icon" viewBox="0 0 77 100"><g fill="none" fill-rule="evenodd"><path stroke="currentColor" stroke-width="7.5" d="M38.794 93.248C58.264 67.825 68 49.692 68 38.848 68 22.365 54.57 9 38 9S8 22.364 8 38.85c0 10.842 9.735 28.975 29.206 54.398a1 1 0 0 0 1.588 0z"></path><circle cx="38" cy="39" r="10" fill="currentColor"></circle></g></svg>
+				' . esc_html( $location ) . '</span>
+			</div>
+			</div>';
+
+	}
+
+	$output .= '</div>
+			</div>
+		</div>
+	</section>';
+
+	return $output;
+}
+
+/**
+ * Enqueues styles.
+ */
+function amp_travel_enqueue_styles() {
+	if ( is_singular( 'adventure' ) ) {
+		wp_enqueue_style( 'amp_travel_adventure', get_template_directory_uri() . '/assets/css/adventure.css' );
+	}
+}
+add_action( 'wp_enqueue_scripts', 'amp_travel_enqueue_styles' );
+
+/**
+ * Add rating field to comments.
+ */
+function amp_travel_comment_rating_field() {
+	echo '<p class="comment-form-rating"><label for="rating">' . esc_html( 'Your rating' ) . '</label>
+			<select required="required" class="select-arr rounded" name="rating" id="rating" [disabled]="commentform_post_' . esc_attr( get_the_ID() ) . '.submitting" on=\'change:AMP.setState( { commentform_post_' . esc_attr( get_the_ID() ) . ': { values: { "rating": event.value } } } )\'>
+			<option value="">--</option>';
+
+	for ( $i = 5; $i >= 1; $i-- ) {
+		/* translators: %d: Rating */
+		echo '<option value="' . esc_attr( $i ) . '">' . sprintf( _n( '%d star', '%d stars', $i, 'travel' ), number_format_i18n( $i ) ) . '</option>';
```

Comment:
@kienstra This line is now throwing CS error:
`includes/functions.php:153:67: error - All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '_n'. (WordPress.XSS.EscapeOutput.OutputNotEscaped)`.

Any thoughts how to fix that?

Reply:
Hi @miina, wrapping the `_n()` call in `esc_html()` should take care of that.